### PR TITLE
Feature/add review delete

### LINKED ISSUE #354: [queue] Button to Delete a branch, cancel the request
Give the administrator the ability to delete the branch without merging it into master.. effectively denying the request for modification, and giving the ability to maintain the queue/branches

### PR DESCRIPTION
**Pull request by lsat12357**
fixes #354

### CODE REVIEW

**Comment by lsat12357** [comment]
capture1

File: app/controllers/review_controller.rb [hunk 1]
```diff
@@ -36,6 +36,16 @@ def edit
     end
   end
 
+  def discard
+    rugged_delete_branch(params[:id])
+    begin
```
Comment (straleyb) [inline app/controllers/review_controller.rb]: Maybe if there is a begin we could have a rescue? Just in case it throws an error.

Comment (lsat12357) [reply app/controllers/review_controller.rb]: heh. unfortunately, if the term does not exist, the response IS an error.

Comment (lsat12357) [reply app/controllers/review_controller.rb]: irb(main):003:0> Term.exists? "bleha/foo"
[INFO] fetch_cached_graph(http://opaquenamespace.org/ns/bleha/foo) not found in triplestore cache (http://blazegraph:9999/blazegraph/namespace/development/sparql)
[ERROR] TriplestoreRepository.statements failed with TriplestoreException: graph not found in cache, from_remote:, skipping fetch from remote graph
=> false

Comment (straleyb) [reply app/controllers/review_controller.rb]: flash[:notice] = Term.exists? params[:id] ? "Modifications to #{params[:id]} are discarded." : "#{params[:id]} has been discarded."

**Pull request closed without merging**